guide/startdiscussion.md

```diff
@@ -0,0 +1,30 @@
+## User Handbook
+### Starting a Discussion
+
+Discussions can be started once you are logged in, to begin click on **Start a Discussion** to the left hand side of your screen. The button for this looks like this:
+
+![SS - Start a discussion button](687474703a2f2f692e696d6775722e636f6d2f3273757538696c2e706e67.png)
+
+Once that button is pressed, at the bottom center of your screen, the discussion dialogue box will appear so you can start your discussion, the box looks like this:
+
+![SS - Dialogue box bottom](687474703a2f2f692e696d6775722e636f6d2f495049527356512e706e67.png)
+
+### Form Components
+
+### Tags
+
+On the form, you'll need to select some tags to categorise what you are talking about. Click on "Choose Tags" from the screenshot above, and the following page will display:
+
+![SS - Tag screen](687474703a2f2f692e696d6775722e636f6d2f5662794e6e375a2e706e67.png)
+
+Select the tags you think are appropriate, or relevant to your discussion, and then click "Ok" to apply them.
+
+### Discussion Title
+
+Discussion Title is giving a very short, brief subject on your discussion, for example if you wanted to start a discussion about the weather today, then your Title would be something along the lines of "The Weather Today".
+
+### Write a post
+
+This is where you expand in more detail about what you want to talk about, feel free to go crazy here! (although, not too crazy!).
```

Comment:
(although, not too crazy!)

Like they should follow the rules of their community? ;)